src/book/entities/Account.ts

```diff
@@ -134,11 +134,12 @@ export default class Account extends BaseEntity {
   }
 
   async remove(options?: SaveOptions): Promise<this> {
+    const account = await super.remove(options);
+
     if (this.queryClient) {
       updateCache({ queryClient: this.queryClient });
     }
 
-    const account = await super.remove(options);
```

Comment:
I'm not sure why it was in this order but this creates an issue where the cache is updated before it is deleted.